models/experimental/functional_unet/tt/unet_shallow_ttnn.py

```diff
@@ -413,6 +435,7 @@ def __init__(self, parameters: ParameterDict, device, mesh_mapper=None) -> None:
             conv_cache=self.conv_cache,
             should_reshard=True,
             mesh_mapper=mesh_mapper,
+            nhw_core_override=60,
```

Comment:
This has the same effect as modifying this from `main`? https://github.com/tenstorrent/tt-metal/blob/b4d605f5fb27c7a43a6c6ad25a38af51853a2ded/ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp#L99